integration/spark2/src/main/scala/org/apache/spark/sql/hive/CarbonAnalysisRules.scala

```diff
@@ -49,10 +49,26 @@ case class CarbonIUDAnalysisRule(sparkSession: SparkSession) extends Rule[Logica
 
       val projList = Seq(UnresolvedAlias(UnresolvedStar(alias.map(Seq(_)))), tupleId)
 
+      val tableRelation = if (sparkSession.version.startsWith("2.1")) {
```

Comment:
use org.apache.spark.SPARK_VERSION instead of using sparkSession object

Reply:
Ok.